opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/interactor/LDActiveManagementStageActivityInteractor.java

```diff
@@ -197,7 +197,7 @@ public String evaluateSubTitle() {
         public BaseLDVisitAction.Status evaluateStatusOnPayload() {
             if (StringUtils.isNotBlank(placenta_and_membrane_expulsion) && StringUtils.isNotBlank(estimated_blood_loss)) {
                 return BaseLDVisitAction.Status.COMPLETED;
-            } else if (StringUtils.isBlank(estimated_blood_loss)) {
+            } else if (StringUtils.isNotBlank(placenta_and_membrane_expulsion) && StringUtils.isBlank(estimated_blood_loss)) {
```

Comment:
Refactor this condition to

`StringUtils.isNotBlank(placenta_and_membrane_expulsion) || StringUtils.isNotBlank(estimated_blood_loss)`

Reply:
@cozej4 Alright, on it!